packages/package_info_plus/package_info_plus/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.4.2
```

Comment:
is it missing the respective version bump in `pubspec.yaml`?

Reply:
Woops, thanks for the catch !